tests/yamitranscode.cpp

```diff
@@ -43,7 +43,11 @@ static void print_help(const char* app)
     printf("   -N <number of frames to encode(camera default 50), useful for camera>\n");
     printf("   -t <AVC scalability temporal layer number  (default 1)> optional\n");
     printf("   --qp <initial qp> optional\n");
-    printf("   --rcmode <CBR|CQP> optional\n");
+    printf("   --rcmode <CBR|CQP|VBR> optional\n");
+    printf("   --tpercentage <target percentage of bitrate in VBR mode, default 95, range in(50-100)> optional\n");
```

Comment:
okay thanks @xuguangxin, I will modify. But I think of it's a little bit too long for a command parameter.